models/generators.py

```diff
@@ -104,73 +107,48 @@ def __init__(self, img_size, num_latent_dims, kernel_size, num_filters,
-        self.bn1 = BatchNormalization(gamma_initializer=self.bn_init)
+        self.dense_1 = Dense(units=128*8*4*4, kernel_initializer=self.w_init)
+        self.bn_1 = BatchNormalization(gamma_initializer=self.bn_init)
         self.reshape_layer = Reshape([4, 4, 128*8])
-        self.relu1 = ReLU()
 
-        self.deconv2d1 = Conv2DTranspose(
-            128*4, kernel_size=(4, 4), strides=(2, 2), padding='same', kernel_initializer=self.w_init
-        )
-        self.conv1 = Conv2D(
-            filters=128*4, kernel_size=(3, 3), strides=(1, 1), padding='same', kernel_initializer=self.w_init
-        )
-        self.bn2 = BatchNormalization(gamma_initializer=self.bn_init)
-        self.relu2 = ReLU()
+        self.res_block_1 = ResidualLayer(128*2, 128*8, self.w_init, self.bn_init)
+        self.relu_1 = ReLU()
 
-        self.deconv2d2 = Conv2DTranspose(
-            128*2, kernel_size=(4, 4), strides=(2, 2), padding='same', kernel_initializer=self.w_init
-        )
-        self.conv2 = Conv2D(
-            filters=128*2, kernel_size=(3, 3), strides=(1, 1), padding='same', kernel_initializer=self.w_init
-        )
-        self.bn3 = BatchNormalization(gamma_initializer=self.bn_init)
-        self.relu3 = ReLU()
+        self.deconv_block_1 = DeconvBlock(128*4, self.w_init, self.bn_init, activation=False)
 
-        self.deconv2d3 = Conv2DTranspose(
-            128, kernel_size=(4, 4), strides=(2, 2), padding='same', kernel_initializer=self.w_init
-        )
-        self.conv3 = Conv2D(
-            filters=128, kernel_size=(3, 3), strides=(1, 1), padding='same', kernel_initializer=self.w_init
-        )
-        self.bn4 = BatchNormalization(gamma_initializer=self.bn_init)
-        self.relu4 = ReLU()
+        self.res_block_2 = ResidualLayer(128, 128*4, self.w_init, self.bn_init)
+        self.relu_2 = ReLU()
 
-        self.deconv2d4 = Conv2DTranspose(
-            num_output_channels, kernel_size=(4, 4), strides=(2, 2), padding='same', kernel_initializer=self.w_init
-        )
-        self.conv4 = Conv2D(
-            filters=num_output_channels, kernel_size=(3, 3), strides=(1, 1),
-            padding='same', kernel_initializer=self.w_init
-        )
+        self.deconv_block_2 = DeconvBlock(128*2, self.w_init, self.bn_init, activation=True)
+        self.deconv_block_3 = DeconvBlock(128, self.w_init, self.bn_init, activation=True)
+
+        self.deconv2d_4 = Conv2DTranspose(num_output_channels, kernel_size=(4, 4), strides=(2, 2), padding='same', kernel_initializer=self.w_init)
+        self.conv2d_4 = Conv2D(filters=num_output_channels, kernel_size=(3, 3), strides=(1, 1), padding='same', kernel_initializer=self.w_init)
 
     @tf.function
     def call(self, embedding, noise):
         smoothed_embedding, mean, log_sigma = self.conditional_augmentation(embedding)
-        noisy_embedding = tf.concat([smoothed_embedding, noise], 1)
+        noisy_embedding = tf.concat([noise, smoothed_embedding], 1)
```

Comment:
Does the order here matter? I'm OK with the change, but I'm just not sure if I'm missing something

Reply:
no different, just OCD haha